wenet/transformer/label_smoothing_loss.py

```diff
@@ -48,11 +48,12 @@ def __init__(self,
                  normalize_length: bool = False):
         """Construct an LabelSmoothingLoss object."""
         super(LabelSmoothingLoss, self).__init__()
-        self.criterion = nn.KLDivLoss(reduction="none")
+        self.size = size
```

Comment:
KLD is not support on IPU?

Reply:
we have rewritten a label smoothing loss, and they are tested to be equal,
the original code's scatter op have some problem in performance.